src/aggregators/directcr.jl

```diff
@@ -87,11 +85,11 @@ function initialize!(p::DirectCRJumpAggregation, integrator, u, params, t)
     # initialize rates
     fill_rates_and_sum!(p, u, params, t)
 
-    # if no maxrate was set, use largest initial rate (pad by 2 for an extra group)
-    isinf(p.maxrate) && (p.maxrate = 2*maximum(p.cur_rates))
-
     # setup PriorityTable
-    p.rt   = PriorityTable(p.ratetogroup, p.cur_rates, p.minrate, p.maxrate)
```

Comment:
How does the initial table change here? Is it the same (i.e. the same set of buckets)?

Reply:
Yes, the same set of buckets. If these buckets were created in the first run, chances are the same buckets will be needed for other runs. So the idea is to just empty all the buckets and fill them again instead of recreating the whole table